drivers/usb/device/usb_dc_stm32.c

```diff
@@ -217,14 +217,35 @@ static int usb_dc_stm32_clock_enable(void)
 		return -ENODEV;
 	}
 
-#if defined(PWR_USBSCR_USB33SV) || defined(PWR_SVMCR_USV)
+#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32_otghs) && defined(CONFIG_SOC_SERIES_STM32U5X)
+	/* Sequence to enable the power of the OTG HS on a stm32U5 serie : Enable VDDUSB */
+	if (__HAL_RCC_PWR_IS_CLK_DISABLED()) {
+		__HAL_RCC_PWR_CLK_ENABLE();
```

Comment:
For consistency, can we use LL API in stead of __HAL_RCC macros ?

Reply:
yes, possible with LL_AHB3_GRP1_IsEnabledClock()